### DISCUSSION TITLE
GitHub Discussions is now open for business!

### DISCUSSION THREAD

**Discussion by Tantacrul** [musescore/MuseScore#11537] [Announcements]
We've opened this up so that community members can raise topics that we can keep track of without the rigidity of the 'issue' system.

Enjoy!

**Comment by CyberFlameGO** [comment]
Sweet!

**Comment by jivy59** [comment]
Will the "unroll" function be available when 4.0 is released? It is very helpful for making demo videos for my publisher.

**Reply by Tantacrul** [reply]
What is an unroll feature?

**Reply by MarcSabatella** [reply]
MuseScore 3 had a command Tools / Unroll Repeats, which created a new score that was cloned from the current score but with the repeats written out in full.  So an 8 bar piece with an end repeat would be turned into a 16 bar piece with bars 9-16 a duplicate of 1-8.  It handled voltas and DS.  Useful especially for things like crafting playback when you wanted a given passage played second time only or whatever.  Potentially less important if features like that ever get implemented directly, but probably use cases still exist.

**Reply by Tantacrul** [reply]
Ah, unroll **repeats**. Got it.

**Comment by markgalassi** [comment]
Dear MuseScore developers, this is not a technical question, but might spawn good contributions:

At the Institute for Computing in Research we will be paying a young intern from Austin to work full time for a month, and he wants to work on software development for MuseScore.  (He's a musician and future computer scientist and passionate MuseScore users.)

Y'all have done a very good job of documentation for contributors, but I wonder if any experienced developer might want to go slightly farther and be available to talk to me and this student about the best strategy for him to embark on this work.  (You are certainly not signing up to help him write code at all; just to provide intuition and strategy.)

If you have tips for me, or can talk to me about MuseScore development, please call me or email me.

Mark Galassi
mark@galassi.org
+1-505-629-0759 (voice only)
https://www.galassi.org/mark/

**Reply by Tantacrul** [reply]
Hey Mark, I'm the head of product at Muse. Send me a mail at info.tantacrul@gmail.com and I'll set you up with the right people.
We're always happy to guide contributors and provide support for them.
